gensim/utils.py

```diff
@@ -1260,7 +1264,8 @@ def chunkize(corpus, chunksize, maxsize=0, as_numpy=False):
 
         """
         if maxsize > 0:
-            warnings.warn("detected Windows; aliasing chunkize to chunkize_serial")
+            entity = "Windows" if os.name == 'nt' else "OSX with python3.8+"
+            warnings.warn("detected {entity}; aliasing chunkize to chunkize_serial".format(entity=entity))
```

Comment:
I think there is a preference to avoid .format in gensim code.

```suggestion
            warnings.warn("detected %s; aliasing chunkize to chunkize_serial" % entity)
```

Reply:
So, strange preference, you moving forward to get rid all stuff <3.6, but still using `%` formatting

```bash
ivan@P50:~/release/gensim$ find gensim -name "*.py" | xargs grep "\.format(" | wc -l
83
ivan@P50:~/release/gensim$ find gensim -name "*.py" | xargs grep "% (" | wc -l
139  # not quiet, this number will be larger in reality
```

but ok, up to you